charts/t8s-cluster/templates/management-cluster/clusterClass/clusterClass.yaml

```diff
@@ -0,0 +1,148 @@
+spec:
+  controlPlane:
+    machineHealthCheck:
+      maxUnhealthy: 33%
+      nodeStartupTimeout: 10m
+      unhealthyConditions:
+        - status: Unknown
+          timeout: 600s
+          type: Ready
+        - status: 'False'
+          timeout: 600s
+          type: Ready
+    nodeDrainTimeout: {{ .Values.controlPlane.nodeDrainTimeout | default "8m" }}
+    machineInfrastructure:
+      ref:
+        apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
+        kind: OpenStackMachineTemplate
+        name: {{ printf "%s-control-plane-%s" $.Release.Name (include "t8s-cluster.clusterClass.openStackMachineTemplate.specHashOfControlPlane" (dict "context" $)) }}
+    ref:
+      apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+      kind: KubeadmControlPlaneTemplate
+      # the full context is needed for .Files.Get
+      name: {{ printf "%s-%s" $.Release.Name (include "t8s-cluster.clusterClass.kubeadmControlPlaneTemplate.specHash" .) }}
+  infrastructure:
+    ref:
+      apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
+      kind: OpenStackClusterTemplate
+      name: {{ printf "%s-%s" $.Release.Name (include "t8s-cluster.clusterClass.openStackClusterTemplate.specHash" (dict "context" $)) }}
+  patches:
+    - definitions:
+        - jsonPatches:
+            - op: add
+              path: /spec/template/spec/serverGroupID
+              valueFrom:
+                variable: controlPlaneServerGroupID
+          selector:
+            apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
+            kind: OpenStackMachineTemplate
+            matchResources:
+              controlPlane: true
+      description: Sets the ServerGroupID for Control Plane machines.
+      name: controlPlaneServerGroupID
+    - definitions:
+        - jsonPatches:
+            - op: add
+              path: /spec/template/spec/serverGroupID
+              valueFrom:
+                variable: machineDeploymentServerGroupID
+          selector:
+            apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
+            kind: OpenStackMachineTemplate
+            matchResources:
+              machineDeploymentClass:
+                names: {{- .Values.workers | keys | sortAlpha | toYaml | nindent 18 }}
+      description: Sets the ServerGroupID for MachineDeployment machines.
+      name: machineDeploymentServerGroupID
+    - name: openStackCloudConfiguration
+      definitions:
+        - selector:
+            apiVersion: infrastructure.cluster.x-k8s.io/v1alpha5
+            kind: OpenStackClusterTemplate
+            matchResources:
+              infrastructureCluster: true
+          jsonPatches:
+            - op: add
+              path: /spec/template/spec/controlPlaneAvailabilityZones
+              valueFrom:
+                variable: controlPlaneAvailabilityZones
+            - op: add
+              path: /spec/template/spec/dnsNameservers
+              valueFrom:
+                variable: dnsNameservers
```

Comment:
If the dnsNameservers will be handled by the operator, this is not needed, and should be removed. Or am i wrong`

Reply:
Ok, this is set via the operator. Good like this.